src/MizarData/MizarPairedDataTest.cpp

```diff
@@ -70,8 +69,20 @@ constexpr uint64_t kRelativeTime2 = 15;
 constexpr uint64_t kRelativeTime3 = 20;
 constexpr uint64_t kRelativeTime4 = 30;
 constexpr uint64_t kRelativeTime5 = 40;
-constexpr uint64_t kTID = 0x3AD1;
-constexpr uint64_t kAnotherTID = 0x3AD2;
+constexpr uint64_t kRelativeTimeTooLate = 1000;
+constexpr uint32_t kTID = 0x3AD1;
+constexpr uint32_t kAnotherTID = 0x3AD2;
+constexpr uint32_t kNamelessTID = 0x3AD3;
+const std::string kThreadName = "thread";
+const std::string kOtherThreadName = "other thread";
```

Comment:
Nit: It's good practice to wrap the literals into a std::string_view and construct the strings when needed through the explicit constructor.